setup.py

```diff
@@ -17,7 +17,6 @@
         "console_scripts": ["clr = clr:main"],
     },
     install_requires=requirements,
-    setup_requires=["pytest-runner"],
```

Comment:
This is a bad PR. It fixes build reproducibility, but breaks `python setup.py test`. That said, the latter is known broken. See the deprecation notice here: https://pypi.org/project/pytest-runner/

Best seems to be to follow those recommendations and add a tox.ini or your preferred choice venv manager to use to run pytest.